lib/services/components.test.js

```diff
@@ -666,12 +682,28 @@ describe(_.startCase(filename), function () {
       expect(fn('name', 'slug')).to.deep.equal(['/css/name.css', '/css/name.slug.css']);
     });
 
-    it('accepts good component with slug (with slug file) with assetDir and assetPath', function () {
+    it('sometimes accepts good component with slug (with slug file) with assetDir and assetPath', function () {
       siteService.sites.returns({slug: {assetDir: 'someAssetDir', assetPath: '/someAssetPath'}});
       files.fileExists.withArgs('someAssetDir/css/name.css').returns(true);
       files.fileExists.withArgs('someAssetDir/css/name.slug.css').returns(true);
 
       expect(fn('name', 'slug')).to.deep.equal(['/someAssetPath/css/name.css', '/someAssetPath/css/name.slug.css']);
     });
+
+    it('accepts good component with slug (with slug file) with assetDir and assetPath and path', function () {
+      siteService.sites.returns({slug: {assetDir: 'someAssetDir', path: '/scienceofus', assetPath: '/scienceofus'}});
+      files.fileExists.withArgs('someAssetDir/css/name.css').returns(true);
+      files.fileExists.withArgs('someAssetDir/css/name.slug.css').returns(true);
+
+      expect(fn('name', 'slug')).to.deep.equal(['/scienceofus/css/name.css', '/scienceofus/css/name.slug.css']);
+    });  
+
+    it('accepts good component with slug (with slug file) with assetDir and assetPath and path', function () {
+      siteService.sites.returns({slug: {assetDir: 'someAssetDir', path: '/', assetPath: '/vulture'}});
+      files.fileExists.withArgs('someAssetDir/css/name.css').returns(true);
+      files.fileExists.withArgs('someAssetDir/css/name.slug.css').returns(true);
+
+      expect(fn('name', 'slug')).to.deep.equal(['/css/name.css', '/css/name.slug.css']);
+    });   
```

Comment:
This is a good test because it shows what the expectations are.

```
assetDir: 'someAssetDir'
path: '/'
assetPath: '/vulture'
```

With this configuration, I think people would expect the assets to be found at `/vulture`, not `/`.  However, the test is expecting the assets at `/`.

Could we just set the assetPath to `/` instead?